src/document.rs

```diff
@@ -50,6 +54,10 @@ fn value_to_string(value: &Value) -> String {
         Value::Bytes(bytes) => format!("{:?}", bytes),
         Value::Date(d) => format!("{:?}", d),
         Value::Facet(facet) => facet.to_string(),
+        Value::PreTokStr(pretok) => {
+            // TODO implement me
```

Comment:
Can't we just use `pretok.text.clone()` here? I know that this isn't strictly required since we don't have the ability to add them but this seems to be easy to implement?